app/server/lib/ConfigBackendAPI.ts

```diff
@@ -0,0 +1,67 @@
+import * as express from 'express';
+import {expressWrap} from 'app/server/lib/expressWrap';
+
+import {IGristCoreConfig} from 'app/server/lib/configCore';
+import {getGlobalConfig} from 'app/server/lib/globalConfig';
+
+import log from "app/server/lib/log";
+
+export class ConfigBackendAPI {
+  private static _doRestart = false;
```

Comment:
Can you add comments here, about this static thing. It looks very dangerous. We run multiple servers in tests in the same time in memory.

Reply:
I don't like the static variable either, but what would you suggest instead? I used to have a normal instance variable but it turns out that there were multiple `ConfigBackendAPI` objects (not sure how) and the restart never happened.

Reply:
I think I've found what is causing it. The explanation is in the comment below.